modules/cli/pages/cbstats/cbstats-durability-monitor.adoc

```diff
@@ -0,0 +1,109 @@
+= durability-monitor
+:page-topic-type: reference
+
+[abstract]
+Provides durability statistics in relation to vBuckets.
+
+[#syntax]
+== Syntax
+
+Request syntax:
+
+----
+cbstats host:11210 [common options] durability-monitor vbid
+----
+
+[#description]
+== Description
+
+This command provides durability statistics for the specified vBucket.
+Note that the vBucket ID _must_ be specified.
+
+In the displayed output, the identifier for each durability statistic begins with the string `vb_`, followed by the vBucket ID and a colon.
+For example, for vBucket 1023, the identifier for the `high_prepared_seqno` statistic is `vb_1023:high_prepared_seqno`.
+
+For an overview of durability in Couchbase Server, see xref:learn:data/durability.adoc[Durability].
+
+[#durability-monitor-statistics]
+.Durability Monitor Statistics
+[cols="2,3"]
+|===
+| Name | Description
+
+| `high_prepared_seqno`
+| The highest sequence number among the prepared durable writes for the vBucket.
+_Prepared_ durable writes are those in progress, and therefore not yet
+committed or aborted.
+The _highest_ sequence number is that of the durable write that has most recently met its durability requirements on the node for the vBucket.
+The durability requirements depend on the durability level specified for the write.
+Provided for both active and replica vBuckets.
+
+| `last_aborted_seqno`
+| The sequence number of the last aborted durable write for the vBucket.
+Provided for active vBuckets only.
+
+| `last_committed_seqno`
+| The sequence number of the last committed durable write for the vBucket.
+Provided for active vBuckets only.
+
+| `high_completed_seqno`
+| The highest sequence number among the completed durable writes for the vBucket.
+_Completed_ includes both _aborted_ and _committed_.
+Provided for replica vBuckets only.
+
+| `last_tracked_seqno`
+| The sequence number of the last durable write to be tracked for the vBucket.
+Provided for active vBuckets only.
+
+| `num_tracked`
+| The number of durable writes currently being tracked for the vBucket.
+Provided for active vBuckets only.
+
+| `replication_chain_First:ns_1@<node-ip-address>:last_ack_seqno`
+| The sequence number of the last durable write for which a buffer-size acknowledgement was received by the producer from the consumer.
```

Comment:
In this context, "last_ack_seqno" is the last prepared SyncWrite which the given node has acknowledged to the active node - it's not related to DCP buffering. See https://docs.google.com/document/d/1DCIzCNXmm3XjkylF19zwNvxcwJFp3bLqSIsQZzTakZ0/edit#

Reply:
Thanks.